### PR TITLE
Refs 441 update swagger documentation disruption

### PR DESCRIPTION
**Pull request by pthegner**
# Description

This update swagger documentation for category disruption

### CODE REVIEW

File: documentation/swagger.yml [hunk 1]
```diff
@@ -992,30 +1003,22 @@ paths:
           description: Bad Request
           schema:
             type: object
```
Comment (Netmisa) [inline documentation/swagger.yml]: You can remove this line

File: documentation/swagger.yml [hunk 2]
```diff
@@ -992,30 +1003,22 @@ paths:
...
-            properties:
-              tag:
-                $ref: "#/definitions/error_occured"
+            $ref: "#/definitions/error_occured"
         404:
           description: Not found
           schema:
             type: object
```
Comment (Netmisa) [inline documentation/swagger.yml]: You can remove this line

File: documentation/swagger.yml [hunk 3]
```diff
@@ -992,30 +1003,22 @@ paths:
...
-            properties:
-              tag:
-                $ref: "#/definitions/error_occured"
+            $ref: "#/definitions/error_404"
         500:
           description: "Server error"
           schema:
             type: object
```
Comment (Netmisa) [inline documentation/swagger.yml]: You can remove this line

File: documentation/swagger.yml [hunk 4]
```diff
@@ -992,30 +1003,22 @@ paths:
...
-            properties:
-              tag:
-                $ref: "#/definitions/error_occured"
+            $ref: "#/definitions/error_occured"
         503:
           description: "Error during transferring of disruption to Navitia"
           schema:
             type: object
```
Comment (Netmisa) [inline documentation/swagger.yml]: You can remove this line

File: documentation/swagger.yml [hunk 5]
```diff
@@ -1032,16 +1035,17 @@ paths:
           description: Disruption could be find
           schema:
             type: object
```
Comment (Netmisa) [inline documentation/swagger.yml]: You can remove this line

File: documentation/swagger.yml [hunk 6]
```diff
@@ -1032,16 +1035,17 @@ paths:
...
-            properties:
-              tag:
-                $ref: "#/definitions/disruption"
+            $ref: "#/definitions/disruption"
         400:
-          description: "Disruption couldn't be found"
+          description: Disruption bad id
           schema:
             type: object
```
Comment (Netmisa) [inline documentation/swagger.yml]: You can remove this line

File: documentation/swagger.yml [hunk 7]
```diff
@@ -1062,16 +1066,22 @@ paths:
           description: Disruption could be find
           schema:
             type: object
```
Comment (Netmisa) [inline documentation/swagger.yml]: You can remove this line

File: documentation/swagger.yml [hunk 8]
```diff
@@ -1062,16 +1066,22 @@ paths:
...
-            properties:
-              tag:
-                $ref: "#/definitions/disruption"
+            $ref: "#/definitions/disruption"
         400:
-          description: "Disruption couldn't be found"
+          description: Bad request
           schema:
             type: object
```
Comment (Netmisa) [inline documentation/swagger.yml]: You can remove this line

File: documentation/swagger.yml [hunk 9]
```diff
@@ -1062,16 +1066,22 @@ paths:
...
-            properties:
-              error:
-                $ref: "#/definitions/bad_id"
+            $ref: "#/definitions/error_occured"
+        404:
+          description: Not found
+          schema:
+            type: object
```
Comment (Netmisa) [inline documentation/swagger.yml]: You can remove this line

File: documentation/swagger.yml [hunk 10]
```diff
@@ -1062,16 +1066,22 @@ paths:
...
+            $ref: "#/definitions/error_404"
+        409:
+          description: Conflict
+          schema:
+            type: object
```
Comment (Netmisa) [inline documentation/swagger.yml]: You can remove this line

File: documentation/swagger.yml [hunk 11]
```diff
@@ -1032,16 +1035,17 @@ paths:
...
           schema:
             type: object
-            properties:
-              error:
-                $ref: "#/definitions/bad_id"
+            $ref: "#/definitions/error_occured"
+        404:
+          description: Not found
+          schema:
+            type: object
```
Comment (Netmisa) [inline documentation/swagger.yml]: You can remove this line

File: documentation/swagger.yml [hunk 12]
```diff
@@ -1081,17 +1091,20 @@ paths:
...
           schema:
             type: object
-            properties:
-              error:
-                $ref: "#/definitions/bad_id"
+            $ref: "#/definitions/error_occured"
+        503:
+          description: Service unavailable
+          schema:
+            type: object
```
Comment (Netmisa) [inline documentation/swagger.yml]: You can remove this line

File: documentation/swagger.yml [hunk 13]
```diff
@@ -1081,17 +1091,20 @@ paths:
...
+        - Disruption
       responses:
         204:
           description: Disruption has been removed
-        400:
-          description: BAD REQUEST
+        500:
+          description: Internal server error
           schema:
             type: object
```
Comment (Netmisa) [inline documentation/swagger.yml]: You can remove this line

**Comment by kun2985** [comment]
In GET /disruptions the default values of parameters `publication_status`, `current_time` and `status` are not showed in the swagger doc

File: documentation/swagger.yml [hunk 14]
```diff
@@ -1008,36 +1019,23 @@ paths:
...
-                $ref: "#/definitions/error_occured"
+            $ref: "#/definitions/error_occured"
         404:
           description: Not found
           schema:
-            type: object
-            properties:
-              tag:
-                $ref: "#/definitions/error_occured"
+            $ref: "#/definitions/error_404"
```
Comment (kun2985) [inline documentation/swagger.yml]: I get a 404 error with POST /disruptions like:
```
{
    "error": {
        "message": "ptobject a2b10a90-0ce8-11e8-a79e-185e0fa9992d doesn't exist"
    }
}
```

File: documentation/swagger.yml [hunk 15]
```diff
@@ -1053,17 +1051,15 @@ paths:
         201:
           description: Disruption could be find
           schema:
-            type: object
```
Comment (kun2985) [inline documentation/swagger.yml]: I get the following response when I GET /disruptions/{id}:
```
{
    "disruption": {
        "cause": {
            ...
        },
        ...
    }
}
```

File: documentation/swagger.yml [hunk 16]
```diff
@@ -1008,36 +1019,23 @@ paths:
         201:
           description: Created disruption
           schema:
-            type: object
```
Comment (kun2985) [inline documentation/swagger.yml]: I get the following response when I POST /disruptions/:
```
{
    "disruption": {
        "cause": {
            ...
        },
        ...
    }
}
```

File: documentation/swagger.yml [hunk 17]
```diff
@@ -1008,36 +1019,23 @@ paths:
...
           description: "Error during transferring of disruption to Navitia"
           schema:
-            type: object
-            properties:
-              tag:
-                $ref: "#/definitions/error_occured"
+            $ref: "#/definitions/error_occured"
   /disruptions/{id}:
     get:
       description: Retrieve the disruption identified by ID
```
Comment (kun2985) [inline documentation/swagger.yml]: There should have a parameters `depth`: https://github.com/CanalTP/Chaos/blob/master/documentation/disruptions.md#parameters-21

File: documentation/swagger.yml [hunk 18]
```diff
@@ -1083,17 +1079,19 @@ paths:
         201:
           description: Disruption could be find
           schema:
-            type: object
```
Comment (kun2985) [inline documentation/swagger.yml]: I get the following response when I PUT /disruptions/{id}:
```
{
    "disruption": {
        "cause": {
            ...
        },
        ...
    }
}
```

File: documentation/swagger.yml [hunk 19]
```diff
@@ -1083,17 +1079,19 @@ paths:
...
-            properties:
-              error:
-                $ref: "#/definitions/bad_id"
+            $ref: "#/definitions/error_occured"
+        404:
+          description: Not found
+          schema:
+            $ref: "#/definitions/error_404"
+        409:
+          description: Conflict
```
Comment (kun2985) [inline documentation/swagger.yml]: There is no conflict unless for `status`, I think we should add more description here:
https://github.com/CanalTP/Chaos/blob/master/documentation/disruptions.md#but-you-cant-make-a-published-disruption-going-back-to-draft-status

File: documentation/swagger.yml [hunk 20]
```diff
@@ -1103,17 +1101,18 @@ paths:
...
       - $ref: "#/parameters/entity_id"
       tags:
-        - Tag
+        - Disruption
       responses:
         204:
           description: Disruption has been removed
-        400:
-          description: BAD REQUEST
+        500:
```
Comment (kun2985) [inline documentation/swagger.yml]: Add error 400 and 404

**Pull request merged**